README.md

```diff
@@ -197,19 +197,19 @@ Pre-issuance linting is **strongly recommended** by the [Mozilla root
 program](https://wiki.allizom.org/CA/Required_or_Recommended_Practices#Pre-Issuance_Linting).
 Here are some projects/CAs known to integrate with ZLint in some fashion:
 
-* [Camerfirma](https://bugzilla.mozilla.org/show_bug.cgi?id=1556806#c5)
-* [CFSSL](https://github.com/cloudflare/cfssl/pull/1015)
-* [Sectigo and crt.sh](https://groups.google.com/forum/#!msg/mozilla.dev.security.policy/sjXswrcsvrE/Nl3OLd4PAAAJ)
-* [Digicert](https://bugzilla.mozilla.org/show_bug.cgi?id=1550645#c9)
+* [Camerfirma](https://www.camerfirma.com/)
+* [CFSSL](https://github.com/cloudflare/cfssl)
+* [Sectigo](https://sectigo.com/) and [crt.sh](crt.sh)
```

Comment:
Do you need a scheme here for the crt.sh link?

Reply:
Oops, good catch, will fix. Thanks @sleevi!

Reply:
https://github.com/zmap/zlint/pull/349